apps/api/src/app/workflows-v2/workflow.controller.ts

```diff
@@ -90,7 +89,6 @@ export class WorkflowController {
   }
 
   @Put(':workflowId/sync')
-  @UseGuards(UserAuthGuard)
```

Comment:
If there is a `@UserAuthentication()` on top of controller, doing `@UseGuards(UserAuthGuard)` on each method is redundant, because `@UserAuthentication()` already contains that guard inside it